megamek/src/megamek/common/weapons/ArtilleryBayWeaponIndirectFireHandler.java

```diff
@@ -335,7 +335,15 @@ else if (null != bestSpotter) {
             // we do this here to avoid duplicating handle()
             // in the ArtilleryWeaponDirectFireHandler
             Coords origPos = targetPos;
-            targetPos = Compute.scatterDirectArty(targetPos, toHit.getMoS());
+            int moF = -toHit.getMoS();
```

Comment:
The original code uses `toHit.getMoS()` as the parameter passed into `scatterDirectArty`whereas the new version uses `-toHit.getMoS()`.  Was this intentional?